modules/test/playwright/tests/frontend-data-set-views-web/pages/ActionsPage.ts

```diff
@@ -49,9 +49,13 @@ export class ActionsPage {
 
 	async goto() {
 		await this.viewsPage.goto();
-		await this.viewsPage.gotoSampleDataSetView().then(() => {
-			this.page.getByRole('button', {name: 'Actions'}).first().click();
-		});
+		await this.viewsPage.gotoSampleDataSetView();
+
+		await this.page
+			.locator('li')
```

Comment:
Hmm this looks very fragile. Could this be fixed with an explicit `expect`? It would confirm that the correct page is loaded. `expect` on something unique on the page, not necessary that list item. The check can be in `gotoSampleDataSetView`, or here.

This is probably a very common issue, but typically not seen, as after `goto*` is some specific locator.